### ISSUE TITLE
Make `--bun` default, introduce `--node`

### ISSUE THREAD

**Issue by Electroid** [oven-sh/bun#4464]
Currently, when Bun runs a script with a shebang in package.json, it will default to Node. Before 1.0, we should change this so that it defaults to Bun, and introduce `--node` which would allow the script to run using Node instead.

**Comment by birkskyum** [comment]
Doing this before things actually runs with --bun would break a ton of these guides etc

https://bun.sh/guides/ecosystem/nextjs

**Comment by birkskyum** [comment]
And this delay would reach a lot of users as well

- https://github.com/oven-sh/bun/issues/4143

**Comment by Electroid** [comment]
Yep, these are good points. We decided to punt on this, we'll have to consider this in the future.

**Comment by Hebilicious** [comment]
`--node` is a good idea, this could be set as the default until `--bun` is ready for prime time.

**Comment by birkskyum** [comment]
Feels ready for prime time now

**Comment by zoto-ff** [comment]
that would be nice

**Comment by ptrxyz** [comment]
Can we get a setting in package.json or anywhere to make it so that `bun --bun` is default? Maybe that would be a nice workaround for the time being.

**Comment by zoto-ff** [comment]
> Can we get a setting in package.json or anywhere to make it so that `bun --bun` is default? Maybe that would be a nice workaround for the time being.

try this: https://bun.sh/docs/runtime/bunfig#run-bun-auto-alias-node-to-bun

**Comment by ptrxyz** [comment]
I guess this would be a good fix, but sadly it's broken. See https://github.com/oven-sh/bun/issues/11445
